File: curso-python/ex057.py
#faça um programa que leia o sexo de uma pessoa, mas śo aceite os valores "M" ou "F"
sexo = str(input("digite seu sexo [M/F]: ")).strip().upper()[0]
while sexo not in 'MmFf':
    # Caso esteja errado. peça a digitação novamente até ter um valor correto
    sexo = str(input("Dados inválidos! por favor digite seu sexo: ")).strip().upper()[0]
print(f"sexo {sexo} foi registrado com sucesso.")

File: javascript-exercicios/ex057.js
const prompt = require('prompt-sync')();

let sexo = prompt("Digite seu sexo [M/F]: ").trim().toUpperCase();

while (sexo !== 'M' && sexo !== 'F') {
    sexo = prompt("Dados inválidos! Por favor, digite seu sexo [M/F]: ").trim().toUpperCase();
}

console.log(`Sexo ${sexo} foi registrado com sucesso.`);
